### DISCUSSION TITLE
[v6] How to get current PathPattern for analytics like GA

### DISCUSSION THREAD

**Discussion by auwtch** [remix-run/react-router#8261] [Q&A]
It's like history.listen but instead of an active location returns an active path (or PathPattern). So instead of `/ticket/9/client/8` I would like to get `/ticket/:id/client/:id`. Previously, I did this using custom `Route` component and `useMatch(props.path)`.

**Comment by nmabhinandan** [comment]
Did you get any way to do this in v6?

**Comment by ryanflorence** [comment]
Related: #8327
